spec/controllers/errors_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe ErrorsController do
+  render_views
+
+  # Test that each static page is generated correctly
+  Dir.glob(Rails.root.join('app', 'views', 'errors', '*.erb')).each do |file|
+    status_code = file.split('/').last.split('.').first
```

Comment:
I think it would be more declarative if it was just a list of the statuses we wanted covered: 

```
[404, 500, 503]. each do |status_code|...
```